### PR TITLE
vdi_image_create: new case of vdi_image_create

### PR DESCRIPTION
**Pull request by hellohellenmao**
1. Create a image with vdi format

Signed-off-by: Tingting Mao <timao@redhat.com>

ID: 2064158

### CODE REVIEW

**Comment by hellohellenmao** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/3371

**Comment by hellohellenmao** [comment]
1/1) Host_RHEL.m9.u0.vdi.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.vdi_image_create.q35: PASS (2.93 s)

**Comment by hellohellenmao** [comment]
@XueqiangWei @zhencliu Could you please help to take a review? Thanks.

**Comment by XueqiangWei** [comment]
(1/1) Host_RHEL.m9.u0.vdi.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.vdi_image_create.q35: PASS (6.90 s)

File: qemu/tests/vdi_image_create.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from avocado import TestError
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Convert a vdi image.
```
Comment (XueqiangWei) [inline qemu/tests/vdi_image_create.py:9]: This line of comments is redundant. Others LGTM.

Comment (hellohellenmao) [reply qemu/tests/vdi_image_create.py]: Updated. Thanks.

**Comment by XueqiangWei** [comment]
@zhencliu Please help review it. Many thanks.

**Comment by zhencliu** [comment]
hi @hellohellenmao , do we want to verify this on network storage, such as ceph, iscsi?

**Comment by hellohellenmao** [comment]
> hi @hellohellenmao , do we want to verify this on network storage, such as ceph, iscsi?

Hi @zhencliu, I do not think we need it for this format is only used for converting between VDI and RAW in CVN now. At least not for now.

**Comment by zhencliu** [comment]
Talked with owner, need to check qemu version on RHEL8

**Comment by hellohellenmao** [comment]
> Talked with owner, need to check qemu version on RHEL8

Hi @zhencliu. After checked the production bug that there is no the need of VDI in rhel8. So I updated the code accordingly. Take a review again, please.

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.vdi_image_create.q35: PASS (2.28 s)

**Comment by hellohellenmao** [comment]
Hi, @zhencliu could you please help to take a review, thanks.

File: qemu/tests/cfg/vdi_image_create.cfg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- vdi_image_create:
+    only Host_RHEL.m9
```
Comment (zhencliu) [inline qemu/tests/cfg/vdi_image_create.cfg:2]: what if vdi is supported after m9?
do we need to use 'no' instead of 'only' here?

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.vdi_image_create.q35: PASS (2.04 s)

**Comment by hellohellenmao** [comment]
Hi @zhencliu, could you please help to review again? Thanks.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**